TaxCalculator/pom.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we have a parent pom.xml in our coding-solutions which will be a pom module and have this module as a child module ?

Reply:
Done